openmeter/subject/subject.go

```diff
@@ -62,3 +65,37 @@ func (s SubjectKey) Validate() error {
 
 	return nil
 }
+
+// ListAll returns all the subjects.
+// Helper function for listing all customers. Page param will be ignored.
+func ListAll(ctx context.Context, service Service, namespace string, params ListParams) ([]Subject, error) {
+	subjects := []Subject{}
+	limit := 100
+	page := 1
+
+	for {
+		params := params
+		params.Page = pagination.NewPage(page, limit)
+
+		result, err := service.List(ctx, namespace, params)
```

Comment:
The service should accept List without page params and list all. We should only enforce pagination on the httphandler side.

Given subject expands no ent subresources we should be fine.